README.md

```diff
@@ -78,6 +78,7 @@ app.register(cookie, {
   - More sophisticated cookie signing mechanisms can be implemented by supplying an `Object`. Read more about it in [Custom cookie signer](#custom-cookie-signer).
 
 - `parseOptions`: An `Object` to pass as options to [cookie parse](https://github.com/jshttp/cookie#cookieparsestr-options).
```

Comment:
If we are going to vendor the upstream package, we need to pull in the documentation, too.

Reply:
I was going to ask about that..

Should I remove the README change and import the docs in another PR or add it all here?

Reply:
I would just go ahead and fix it in this PR by pulling in the upstream docs.